src/main/java/com/kobylynskyi/graphql/codegen/mapper/TypeDefinitionToDataModelMapper.java

```diff
@@ -132,6 +139,12 @@ private static ParameterDefinition merge(ParameterDefinition typeDef, ParameterD
                 .flatMap(Collection::stream)
                 .forEach(paramDef -> allParameters
                         .merge(paramDef.getName(), paramDef, TypeDefinitionToDataModelMapper::merge));
+
+
+        createUnknownFields(mappingContext).ifPresent(
+                unknownFields->allParameters.put(mappingContext.getUnknownFieldsPropertyName(),unknownFields)
```

Comment:
space？

Reply:
Fixed. I finally managed to install the custom check-style custom configuration. I kept getting the _Property 'scope' does not exist_ exception before downgrading Intellij